contracts/BadgerVotingShare.sol

```diff
@@ -140,6 +140,10 @@ contract BadgerVotingShare {
         After adding all 3 balances we calculate how much BADGER it corresponds to using the pool's reserves.
     */
     function _uniswapBalanceOf(address _voter) internal view returns (uint256) {
+        uint256 total = badger_wBTC_UniV2.totalSupply();
```

Comment:
Since UniV2 burns the initial total supply
I think this is impossible:
https://github.com/Uniswap/v2-core/blob/ee547b17853e71ed4e0101ccfd52e70d5acded58/contracts/UniswapV2Pair.sol#L119-L122

Reply:
Would be possible for new LPs

Reply:
I don't see any risk in adding the change, it just should never have any impact

Reply:
2e7e627